docs/repository/git-repository-size.html

```diff
@@ -0,0 +1,175 @@
+---
+layout: repository
+title: Git Metrics - Repository Size
+permalink: /repository/git-repository-size
+---
+
+<div class="chart-placeholder">
+	<h3>Number of Commits, Trees, Blobs, Tags and Refs</h3>
+	<canvas
+		data-url=".{{ site.dataURL }}/repository/git-repository-size-objects-count.tsv"
+		data-type="history"
+		data-config='{
+			"views":
+			[
+				{
+					"label": "2 m",
+					"tooltip": "Show the last 2 months",
+					"aggregate": false,
+					"series":
+					[
+						"commits",
+						"trees",
+						"blobs",
+						"tags",
+						"refs"
+					],
+					"visibleSeries":
+					[
+						"commits",
+						"trees",
+						"blobs",
+						"tags",
+						"refs"
```

Comment:
These series can have dramatically different ranges. Should we put them into individual charts?

Reply:
I separated counts and bytes into different charts. The charts allow to view selected series by clicking on the legend items, which I thought is better than too many charts. But yeah, worth thinking about splitting it up. Which series would you suggest to separate, e.g. `tags` and `refs` into a separate chart?

Reply:
Do you know if `refs` include `tags`? If they do, then I would even drop `tags`.

Plus, I am not sure if the total "number of trees" and/or "number of blobs" are actionable metrics. Maybe number of trees/files of HEAD in the default branch represents the current performance characteristics of the repo *for the user* better?

In general, I believe we should track all this data but only visualize the most important/actionable subset.

Reply:
@larsxschneider Great feedback! I changed the `git-sizer` report to include all values and now we are flexible in picking what to display in the charts.

The `refs` count includes the `tags`, so I agree that we only need to show the number of `refs`.
I tweaked the charts to show more actionable metrics and included some comments from [your article](http://larsxschneider.github.io/2016/09/21/large-git-repos):
https://steffen.github.io/hubble/repository/git-repository-size?nwo=test-org/large-repo

Great point on the number of trees/files of HEAD in the default branch. `git-sizer` doesn't include that metric, but we can just get it directly from the repo.

Thank you! And Happy Eastern! 🙂🐰